src/models/__test__/card.util.ts

```diff
@@ -29,7 +29,7 @@ export function makeCard<T extends ScryfallCard>(overrides: Partial<T> = {}): T
     scryfall_uri: "<scryfall uri>",
     type_line: "<type line>",
     set: "<set code>",
-    image_uris: makeImageUris(),
+    image_uris: makeImageUris(overrides.name),
```

Comment:
this is so that the image uri does change for the computed property, so the watch on the `imageUri` does trigger.